Library/Formula/nsd.rb

```diff
@@ -24,12 +22,16 @@ def install
       --with-libevent=#{Formula["libevent"].opt_prefix}
     ]
 
-    args << '--enable-root-server' if build.with? 'root-server'
-    args << '--enable-bind8-stats' if build.with? 'bind8-stats'
-    args << '--enable-ratelimit' if build.with? 'ratelimit'
-    args << '--enable-zone-stats' if build.with? 'zone-stats'
+    args << "--enable-root-server" if build.with? "root-server"
+    args << "--enable-bind8-stats" if build.with? "bind8-stats"
+    args << "--enable-ratelimit" if build.with? "ratelimit"
+    args << "--enable-zone-stats" if build.with? "zone-stats"
 
     system "./configure", *args
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{sbin}/nsd", "-v"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!